### DISCUSSION TITLE
Light and Dark themes

### DISCUSSION THREAD

**Discussion by orychalk** [davidsmorais/kuro#96] [Show and tell]
Hello,

I build 2 themes for me, and I share them here :

![image](https://user-images.githubusercontent.com/57261547/232287896-e305ddfc-3da7-4830-9488-5712b0f011db.png)

![image](https://user-images.githubusercontent.com/57261547/232287981-2778939f-8aa3-4212-9ea5-fe2ba8f131dd.png)

**Comment by orychalk** [comment]
Light :
` "theme": {
        "bgPrimary": " #e9e9e9",
        "bgPrimaryTransparent": " #282A3633",
        "bgSecondary": " #fafafa",
        "bgTertiary": " #ffffff",
        "bgHover": " #5294e2",
        "bgHoverSecondary": " #5aa9f0",
        "bgHoverTertiary": " #5294e2",
        "bgActive": " #5aa9f0",
        "bgActiveSecondary": " #cf0003",
        "bgActiveTertiary": " #2663ce",
        "bgSeparator": " #4e536e33",
        "bgBorder": " #4e536e33",
        "bgScrollbar": " #FFFFFF33",
        "fontColorPrimary": " #222222",
        "fontColorSecondary": " #000000",
        "fontColorTertiary": " #777777",
        "fontColorWarning": " #FF5555",
        "fontColorBrand": " #ee6335",
        "fontColorDisable": " #A19F9D",
        "bgBrand": "#ff557f",
        "bgBrandHover": "#f25179",
        "bgBrandHoverSecondary": "#39786f",
        "bgBrandSecondary": "#52a79b",
        "bgBrandDisable": "#303f40",
        "red": " #ff007f",
        "blue": " #1d4b9c",
        "skyblue": " #5555ff",
        "purple": " #9b00e9",
        "green": " #478f6a"
    }
}`

**Comment by orychalk** [comment]
Dark :
`"theme": {
        "bgPrimary": " #4e536e",
        "bgPrimaryTransparent": " #282A3633",
        "bgSecondary": " #333648",
        "bgTertiary": " #4e536e",
        "bgHover": " #3B3E51",
        "bgHoverSecondary": " #474C68",
        "bgHoverTertiary": " #333855",
        "bgActive": " #3B3E51",
        "bgActiveSecondary": " #3F456B",
        "bgActiveTertiary": " #333648",
        "bgSeparator": " #4e536e33",
        "bgBorder": " #4e536e33",
        "bgScrollbar": " #FFFFFF33",
        "fontColorPrimary": " #FFFFFF",
        "fontColorSecondary": " #FFFFFF",
        "fontColorTertiary": " #FFFFFF",
        "fontColorWarning": " #FF5555",
        "fontColorBrand": " #ffaa7f",
        "fontColorDisable": " #A19F9D",
        "bgBrand": "#ff557f",
        "bgBrandHover": "#f25179",
        "bgBrandHoverSecondary": "#39786f",
        "bgBrandSecondary": "#52a79b",
        "bgBrandDisable": "#303f40",
        "red": " #FF5555",
        "blue": " #FFFFFF",
        "skyblue": " #8BE9FD",
        "purple": " #BD93F9",
        "green": " #50FA7B"
    }
}`

**Reply by Cruising0904** [reply]
I tried this no effect. Kuro/config.json file right?
Should I reboot?

**Comment by orychalk** [comment]
Hello @Cruising0904,

No, it's not in this config file.

You must : 
1/ Quit Kuro
2/ Insert and replace theme code into this file : 
`/home/YOUR_USER/.kuro.json`
![image](https://user-images.githubusercontent.com/57261547/233845423-a0a8a1a8-0263-4eac-8f19-cb37a5ec3f9b.png)

3/ Start Kuro
4/ Select in menu, View/Custom Theme

**Reply by SorenHJohansen** [reply]
I do not see any "View/Custom Theme" option? I have also tried Ctrl + S, which did not do anything.

**Comment by davidsmorais** [comment]
Thanks @orychalk , I'll add these to the documentation soon 👌
